File: src/main/java/com/application/lamion/repository/UserRepository.java
package com.application.lamion.repository;

import com.application.lamion.model.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepository extends JpaRepository<User, Long> {
    User findFirstByEmail(String email);
    boolean existsByEmail(String email);
}

File: server/src/main/java/com/application/lamion/repository/UserRepository.kt
package com.application.lamion.repository

import com.application.lamion.model.User
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface UserRepository : JpaRepository<User?, Long?> {
    fun findFirstByEmail(email: String?): User?
    fun existsByEmail(email: String?): Boolean
}
